tests/test_keep_largest_connected_component.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+from parameterized import parameterized
+
+from monai.transforms import KeepLargestConnectedComponent
+
+grid_1 = np.array([[[0, 0, 1, 0, 0], [0, 2, 1, 1, 1], [1, 2, 1, 0, 0], [1, 2, 0, 1, 0], [2, 2, 0, 0, 2]]])
+grid_2 = np.array([[[0, 0, 0, 0, 1], [0, 0, 1, 1, 1], [1, 0, 1, 1, 2], [1, 0, 1, 2, 2], [0, 0, 0, 0, 1]]])
```

Comment:
May I know what's the difference between tests for `grid_1` and `grid_2`?

Reply:
The first one looks like
0, 0, 1, 0, 0
0, 2, 1, 1, 1
1, 2, 1, 0, 0
1, 2, 0, 1, 0
2, 2, 0, 0, 2

Second one looks like
0, 0, 0, 0, 1
0, 0, 1, 1, 1
1, 0, 1, 1, 2
1, 0, 1, 2, 2
0 ,0 ,0 ,0 ,1

The second case have 1's that are not reachable under all connectivity or independent.